### PR TITLE
feat: add Angular Material

### PR DESCRIPTION
**Pull request by Danieliverant**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: [CONTRIBUTING.md](CONTRIBUTING.md#commit-message-format)
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[X] Other... Please describe: Infra
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[X] Yes
[ ] No
```

Won't impact

### CODE REVIEW

**Comment by ronnetzer** [comment]
instead of `feat(global):` you should just do `feat:`

File: package.json [hunk 1]
```diff
@@ -50,7 +52,6 @@
     "@nestjs/schematics": "^9.0.0",
     "@nestjs/testing": "^9.0.0",
     "@nrwl/angular": "15.7.2",
-    "@nrwl/eslint-plugin-nx": "15.7.2",
```
Comment (ronnetzer) [inline package.json:53]: revert

File: package.json [hunk 2]
```diff
@@ -66,7 +67,6 @@
     "eslint": "~8.15.0",
     "eslint-config-prettier": "8.1.0",
     "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-unused-imports": "^2.0.0",
```
Comment (ronnetzer) [inline package.json:69]: revert

File: package.json [hunk 3]
```diff
@@ -17,10 +17,12 @@
   "dependencies": {
     "@angular-devkit/core": "~15.1.0",
     "@angular/animations": "~15.1.0",
+    "@angular/cdk": "^15.1.5",
```
Comment (ronnetzer) [inline package.json:20]: lock versions with ~

File: package.json [hunk 4]
```diff
@@ -17,10 +17,12 @@
...
     "@angular/common": "~15.1.0",
     "@angular/compiler": "~15.1.0",
     "@angular/core": "~15.1.0",
     "@angular/forms": "~15.1.0",
+    "@angular/material": "^15.1.5",
```
Comment (ronnetzer) [inline package.json:25]: same as prev comment

File: apps/cli-gui/src/main.ts [hunk 5]
```diff
@@ -1,33 +1,19 @@
...
-            core.update({
-              projectNames,
-              currentProjectName: projectNames?.[0],
-            })
-          )
-        ),
-      deps: [HttpClient, CoreService],
-      multi: true,
-    },
+    importProvidersFrom(BrowserAnimationsModule),
```
Comment (ronnetzer) [inline apps/cli-gui/src/main.ts:17]: use newer syntax: `provideAnimations()`

**Review by ronnetzer — CHANGES_REQUESTED** [review]

**Review by agroupp — APPROVED** [review]

**Review by ronnetzer — APPROVED** [review]

**Pull request merged**